index.css

```diff
@@ -79,29 +79,31 @@ h2{
 
 .progress-display{
 
-    width: 400px;
     height: 40px;
     border-radius: 50px;
     background: #44354A;
     margin: auto;
 
 }
 .watched{
- display: flex;
- justify-content: center;
- gap:20px;
+    display: flex;
+    justify-content: space-between;
  
 }
-.watched-lable{
+
+.watched div{
```

Comment:
I've been told never to use div in css, but after googling it I can't find anything. Im gonna ask my mentors at work tomorrow and see what they think.